src/components/InventoryV2/BatchDetails.tsx

```diff
@@ -1,15 +1,133 @@
-import { Typography, useTheme } from '@mui/material';
+import { Box, Grid, Typography, useTheme } from '@mui/material';
 import strings from 'src/strings';
 import Card from 'src/components/common/Card';
+import { Batch } from 'src/types/Batch';
+import OverviewItemCard from '../common/OverviewItemCard';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+import { useEffect, useState } from 'react';
+import { NurseryBatchService } from 'src/services';
+import PhotosList from '../common/PhotosList';
+import useSnackbar from 'src/utils/useSnackbar';
+import { Button } from '@terraware/web-components';
+import BatchDetailsModal from './BatchDetailsModal';
+import { BATCH_PHOTO_ENDPOINT } from 'src/services/NurseryBatchService';
 
-export default function BatchDetails(): JSX.Element {
+interface BatchDetailsProps {
+  batch: Batch;
+  onUpdate: () => void;
+}
+
+export default function BatchDetails({ batch, onUpdate }: BatchDetailsProps): JSX.Element {
   const theme = useTheme();
+  const { isMobile } = useDeviceInfo();
+  const snackbar = useSnackbar();
+
+  const overviewCardData = [
+    {
+      title: strings.GERMINATING_QUANTITY,
+      data: batch.germinatingQuantity,
+    },
+    {
+      title: '',
+      data: '',
+    },
```

Comment:
Just for space? Will the space be okay when in mobile?

Reply:
Oh, you are right, I didn't test it in mobile. Let me check